README.md

```diff
@@ -283,11 +290,18 @@ spec:
 
 CCM will detect that `loadBalancerIP` already was set and not try to create a new Equinix Metal Elastic IP.
 
-If the `Service.Spec.LoadBalancerIP` was *not* set, then CCM will use the Equinix Metal API to request a new, facility-specific
-Elastic IP and set it to `Service.Spec.LoadBalancerIP`.
+##### Equinix EIP
```

Comment:
We should link to this section in step 6 of the Metallb loadbalancing details and step 3 of the kube-vip loadbalancing details, both of which say ~ "a new EIP will be created". If we link to this section it will clear up how that EIP will be created.

Reply:
That is a really good point, helps close the loop. I will add that.

Reply:
Done. Thanks again for the good suggestion!